lib/types/dto.ts

```diff
@@ -0,0 +1,7 @@
+import { components } from './api';
+
+export type ProjectDTO = components['schemas']['ProjectDTO'];
+export type EmployeeDTO = components['schemas']['EmployeeDTO'];
+export type CustomerDTO = components['schemas']['CustomerDTO'];
+export type TaskDTO = components['schemas']['TaskDTO'];
+export type TimesheetDTO = components['schemas']['TimesheetDTO'];
```

Comment:
nice

Reply:
I'm also thinking that you could probably drop the `DTO` from the exported type names? They just come from generated types from the backend because there's a need to differentiate between dao's and dto's